src/battle/battle_cursor.c

```diff
@@ -492,7 +492,7 @@ __attribute__((aligned(4))) static const u8 Unk_ov16_02270A08[NELEMS(Unk_ov16_02
     0x4
 };
 
-__attribute__((aligned(4))) static const u8 Unk_ov16_022701EC[2][3] = {
+__attribute__((aligned(4))) static const u8 battleMenuButtonLayout[2][3] = {
```

Comment:
```suggestion
__attribute__((aligned(4))) static const u8 sBattleMenuButtonLayout[2][3] = {
```

same goes for the other static consts below